stub-server/src/main/java/org/zanata/rest/service/MockFileResource.java

```diff
@@ -51,14 +62,27 @@
 public class MockFileResource implements FileResource {
     @Override
     public Response acceptedFileTypes() {
+        Set<String> extensions = new HashSet<>();
+        for (DocumentType docType: ProjectType.getSupportedSourceFileTypes(ProjectType.File)) {
+            extensions.addAll(docType.getSourceExtensions());
+        }
+        for (DocumentType docType: ProjectType.getSupportedSourceFileTypes(
+            ProjectType.Gettext)) {
+            extensions.addAll(docType.getSourceExtensions());
+        }
         StringSet entity = new StringSet("");
-        entity.addAll(ProjectType.getSupportedSourceFileTypes(ProjectType.File));
-        entity.addAll(ProjectType
-                .getSupportedSourceFileTypes(ProjectType.Gettext));
+        entity.addAll(extensions);
```

Comment:
I don't understand what you mean here. Wouldn't less lines of code mean there is less chance of human error?

I can't figure out what the problem would be with using the `StringSet` directly, like this:

```
        StringSet extensions = new StringSet("");
        for (DocumentType docType: ProjectType.getSupportedSourceFileTypes(ProjectType.File)) {
            extensions.addAll(docType.getSourceExtensions());
        }
        for (DocumentType docType: ProjectType.getSupportedSourceFileTypes(
            ProjectType.Gettext)) {
            extensions.addAll(docType.getSourceExtensions());
        }
        return Response.ok(extensions.toString()).build();
```